### ISSUE TITLE
The link is sent, but there is no window

### ISSUE THREAD

**Issue by Andrey917393dj** [m4dEngi/RemotePlayWhatever#152]
I am on version 0.2.7 send a friend a link he can log in and see my game but when I press shift+tab there is no interface remote player I can not give him access to the mouse and keyboard because there is no interface, the game officially does not support remote play how do I fix it?
This is the interface that doesn't appear👇
![image](https://github.com/m4dEngi/RemotePlayWhatever/assets/87417976/dbaba416-76cc-4b54-9275-5bac23abc60b)

**Comment by Scarriaus** [comment]
Yeah, me too. I guess we'll need to wait for another update from developer.

**Comment by Milkvich11** [comment]
Can confirm, doesn't work either. If that happened before, i invited friend to the other game with native remote support, and then settings were saved, and this window was no longer needed. But now they are not saving.

**Comment by cyborgnrw** [comment]
I have the same problem. This window is important because, for example, only one controller is supported in the game The Quarry in couch co-op mode and the controller is passed on virutelly with the help of this window. I hope this will be fixed.

**Comment by BuIlDaLiBlE** [comment]
As a workaround it is possible to download an older Steam Client from [archive.org](https://archive.org/details/@thefighterjetdude) and use it with the latest (0.2.7) version. Note that `steam.cfg` is included in those archived builds, which disables auto-updates for Steam, so keep that in mind.

**Comment by Solfi321** [comment]
When did the build work fine?

**Comment by BuIlDaLiBlE** [comment]
Idk, maybe try June 10th or 7th.

**Comment by Qunisth** [comment]
I downloaded Steam 03292024 build from link above and used with remoteplay 0.2.6 and this works

**Comment by crudyy** [comment]
> I downloaded Steam 03292024 build from link above and used with remoteplay 0.2.6 and this works

But we'll have to uninstall the current Steam client in order to install the previous client version, right?

**Comment by Qunisth** [comment]
> > I downloaded Steam 03292024 build from link above and used with remoteplay 0.2.6 and this works
> 
> But we'll have to uninstall the current Steam client in order to install the previous client version, right?

No, you can unzip the old steam client into desktop folder and use it separately from the original steam client with settings saved, when you've finished playing non-steam game close old client and open the original client

**Comment by Mikaaaay** [comment]
Still not working for me.....

**Comment by Scarriaus** [comment]
> > > I downloaded Steam 03292024 build from link above and used with remoteplay 0.2.6 and this works
> > 
> > But we'll have to uninstall the current Steam client in order to install the previous client version, right?
> 
> No, you can unzip the old steam client into desktop folder and use it separately from the original steam client with settings saved, when you've finished playing non-steam game close old client and open the original client

You mean open an old version steam when you want to play some non-steam game or some non-RPT. game? Then when you finished with the game close an old version one and continue to use lasted version of steam?

**Comment by crudyy** [comment]
Thank you so much @BuIlDaLiBlE and @Qunisth. This is the best solution I can find right now, cheers!

**Comment by Mikaaaay** [comment]
But what if I decide to play tekken 8 which is a steam game? Not sure how to do that as it is installed in the directory of the original steam client.

**Comment by vyznawca** [comment]
Thank you so much @BuIlDaLiBlE  and @Qunisth. Works great!

**Comment by BuIlDaLiBlE** [comment]
> But what if I decide to play tekken 8 which is a steam game? Not sure how to do that as it is installed in the directory of the original steam client.

You can override your current Steam client with the old version and still play all Steam games normally, at least for some time.

**Comment by AlexKLM** [comment]
Another workaround is to use the big picture mode, when in the ingame overlay you can press Ctrl + 2 to bring out the quick access menu, the remote play interface is accessible in there

**Comment by RebelDima** [comment]
> Another workaround is to use the big picture mode, when in the ingame overlay you can press Ctrl + 2 to bring out the quick access menu, the remote play interface is accessible in there

even so the menu is not shown and it doesn’t seem to work for 3 years

**Comment by AlexKLM** [comment]
> > Another workaround is to use the big picture mode, when in the ingame overlay you can press Ctrl + 2 to bring out the quick access menu, the remote play interface is accessible in there
> 
> even so the menu is not shown and it doesn’t seem to work for 3 years

Hmm, I am not sure why its not working for you, it should be the menu on the right side ?
![240714](https://github.com/user-attachments/assets/514c616b-1ff8-4b78-91b3-4520f1b9d308)

**Comment by crudyy** [comment]
> > > Another workaround is to use the big picture mode, when in the ingame overlay you can press Ctrl + 2 to bring out the quick access menu, the remote play interface is accessible in there
> > 
> > 
> > even so the menu is not shown and it doesn’t seem to work for 3 years
> 
> Hmm, I am not sure why its not working for you, it should be the menu on the right side ? ![240714](https://private-user-images.githubusercontent.com/5012392/348511954-514c616b-1ff8-4b78-91b3-4520f1b9d308.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA5MjYwMjIsIm5iZiI6MTcyMDkyNTcyMiwicGF0aCI6Ii81MDEyMzkyLzM0ODUxMTk1NC01MTRjNjE2Yi0xZmY4LTRiNzgtOTFiMy00NTIwZjFiOWQzMDgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDcxNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA3MTRUMDI1NTIyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YjU1MGJkOWEyMjFlMGJlNzI5MmFhMmJmYzM2YjBhMzg0MWYwYWQ4NzdiYzdmNDMwZTVmY2E3OTNlZmI5ZGVkZCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.6bqlfnlo80E9iwG1CXu9lYUkDRzcO0Oc8Tjt__Dtyws)

I've tried it on a non-steam game. There seems to be no way to change keyboard and mouse access or even change the quality. I can only successfully reorder the controllers, that's it.

**Comment by AlexKLM** [comment]
Ah maybe it could be down to what non steam games you are running? I tried it with nolf2 and can enable keyboard and mouse controls for other players

**Comment by 10XL** [comment]
> > > > I downloaded Steam 03292024 build from link above and used with remoteplay 0.2.6 and this works
> > > 
> > > 
> > > But we'll have to uninstall the current Steam client in order to install the previous client version, right?
> > 
> > 
> > No, you can unzip the old steam client into desktop folder and use it separately from the original steam client with settings saved, when you've finished playing non-steam game close old client and open the original client
> 
> You mean open an old version steam when you want to play some non-steam game or some non-RPT. game? Then when you finished with the game close an old version one and continue to use lasted version of steam?

This might be superfluous but maybe it will help someone. I created a 2gb VHD for that steam client version. Then in steam storage settings I pointed to the current steam folder on my C drive, or wherever your steam is installed. That way I can use my steam library(with games already installed) while being on the same drive.

**Comment by azalty** [comment]
> Another workaround is to use the big picture mode, when in the ingame overlay you can press Ctrl + 2 to bring out the quick access menu, the remote play interface is accessible in there

Thank you very much for this! Worked great :)

Works for windowed games/apps that aren't compatible with the Steam overlay, really nice

**Comment by AtilioA** [comment]
It didn't appear for me the first time I tried it either, but in the subsequent tries I was able to see it within Steam overlay

**Comment by azalty** [comment]
> I've tried it on a non-steam game. There seems to be no way to change keyboard and mouse access or even change the quality. I can only successfully reorder the controllers, that's it. 

Start the game and send an invite link to generate a remote play session. Then, (alt+tab out of the game if needed) start big picture mode in Steam and press CTRL+2. You should be able to change the settings there. Once that's done, you can close or alt+tab big picture and go back to the game.
